autogpt_platform/backend/backend/blocks/smart_decision_maker.py

```diff
@@ -264,9 +263,7 @@ def _create_block_function_signature(
         Raises:
```

Comment:
This needs updated i think

Reply:
nope, technically still valid as that's what `Node.block` does